BALSAMIC/commands/report/deliver.py

```diff
@@ -22,50 +29,21 @@
 LOG = logging.getLogger(__name__)
 
 
-@click.command(
-    "deliver",
-    short_help="Creates a YAML file with output from variant caller and alignment.",
-)
-@click.option(
-    "--sample-config",
-    "-s",
-    required=True,
-    help="Sample config file. Output of balsamic config sample",
-)
-@click.option(
-    "-r",
-    "--rules-to-deliver",
-    multiple=True,
-    help=f"Specify a rule to deliver. Delivery mode selected via --delivery-mode option."
-    f"Current available rules to deliver are: {', '.join(DELIVERY_RULES)} ",
-)
-@click.option(
-    "-m",
-    "--delivery-mode",
-    type=click.Choice(["a", "r"]),
-    default="a",
-    show_default=True,
-    help="a: append rules-to-deliver to current delivery options. "
-    "r: reset current rules to delivery to only the ones specified",
-)
-@click.option(
-    "--disable-variant-caller",
-    help=f"Run workflow with selected variant caller(s) disable. Use comma to remove multiple variant callers. Valid "
-    f"values are: {list(VCF_DICT.keys())}",
-)
+@click.command("deliver", short_help="Creates a JSON report file with output files")
```

Comment:
Is this a typo or has it changed from a YAML to a JSON? 👀 

```suggestion
@click.command("deliver", short_help="Creates a YAML report file with output files")
```

Reply:
The .hk files are actually JSONs. But it's true that we also generate a YAML file. I'll make it more general 👍